### PR TITLE
Revert string centralization (2020-03-18)

### PR DESCRIPTION
**Pull request by ZhilkinSerg**
#### Summary

```SUMMARY: None```

#### Describe the solution

Moved strings from shared header to files where they are actually used.

#### Testing

It compiles.

### CODE REVIEW

File: src/item.cpp [hunk 1]
```diff
@@ -87,6 +87,24 @@
...
+
+static const skill_id skill_cooking( "cooking" );
+static const skill_id skill_melee( "melee" );
+static const skill_id skill_survival( "survival" );
+static const skill_id skill_unarmed( "unarmed" );
+static const skill_id weapon_skill( "weapon" );
+
+static const quality_id qual_JACK( "JACK" );
+static const quality_id qual_LIFT( "LIFT" );
+static const species_id ROBOT( "ROBOT" );;
```
Comment (kevingranade) [inline src/item.cpp]:
```suggestion
static const species_id ROBOT( "ROBOT" );
```

File: src/npc.cpp [hunk 2]
```diff
@@ -65,6 +65,17 @@
...
 
+static const skill_id skill_archery( "archery" );
+static const skill_id skill_barter( "barter" );
+static const skill_id skill_bashing( "bashing" );
+static const skill_id skill_cutting( "cutting" );
+static const skill_id skill_pistol( "pistol" );
+static const skill_id skill_rifle( "rifle" );
+static const skill_id skill_shotgun( "shotgun" );
+static const skill_id skill_smg( "smg" );
+static const skill_id skill_stabbing( "stabbing" );;
```
Comment (kevingranade) [inline src/npc.cpp]:
```suggestion
static const skill_id skill_stabbing( "stabbing" );
```

**Comment by kevingranade** [comment]
Looks like you missed updating tests, compile is failing in tests/item_tname_test.cpp

**Comment by ZhilkinSerg** [comment]
Should be good now.

**Comment by ZhilkinSerg** [comment]
There are will be more follow-ups:

https://github.com/CleverRaven/Cataclysm-DDA/compare/ZhilkinSerg:revert-centralized-strings-2020-03-18...ZhilkinSerg:revert-centralized-strings-2020-03-19?expand=1

https://github.com/CleverRaven/Cataclysm-DDA/compare/ZhilkinSerg:revert-centralized-strings-2020-03-19...ZhilkinSerg:revert-centralized-strings-2020-03-20?expand=1

**Pull request merged**